### PR TITLE
bpo-40939: Restore some stable API functions incorrectly deleted

### PR DESCRIPTION
**Pull request by pablogsal**
https://bugs.python.org/issue40939

### CODE REVIEW

**Review by gvanrossum — COMMENTED** [review]
Shouldn't you also get rid of PyCompileString on line 1241? Because that's just a typo.

**Comment by pablogsal** [comment]
> Shouldn't you also get rid of PyCompileString on line 1241? Because that's just a typo.

Done in 4989ebe

File: Python/pythonrun.c [hunk 1]
```diff
@@ -1236,12 +1236,6 @@ Py_CompileStringExFlags(const char *str, const char *filename_str, int start,
 }
 
 /* For use in Py_LIMITED_API */
```
Comment (gvanrossum) [inline Python/pythonrun.c]: I think this comment should also go -- it looks like it only applies to the (now deleted) function.

Comment (pablogsal) [reply Python/pythonrun.c]: Oh, good point. I was confused by the diff and though that applied to the restored functions. FIxed in e27e218

**Review by gvanrossum — APPROVED** [review]
Have you double checked that all these signatures match? (I haven't, I just trust that you know what you're doing. :-)

**Comment by pablogsal** [comment]
> Have you double checked that all these signatures match?

I have restored them as they were before the removal (so they should match) and then I checked that all of them appear in `Include/pythonrun.h` and that the signatures are correct.

**Comment by gvanrossum** [comment]
Sounds good then -- thanks for doing the due diligence!

**Pull request merged**